pando/http/request.py

```diff
@@ -102,26 +102,15 @@ def make_franken_uri(path, qs):
 
 
 def make_franken_headers(environ):
-    """Takes a WSGI environ, returns a bytestring.
-    """
+    """Takes a WSGI environ, returns a dict of HTTP headers.
 
-    # There are a couple keys that CherryPyWSGIServer explicitly doesn't
-    # include as HTTP_ keys. I'm not sure why, but I believe we want them.
```

Comment:
I think we should keep this comment, or one like it.

@Changaco Do you know why these two headers aren't prefixed with `HTTP_`?